newIDE/app/src/Hints/getTutorial.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import { type Tutorial } from '../Utils/GDevelopServices/Tutorial';
+import { type Preferences } from '../MainFrame/Preferences/PreferencesContext';
+
+/**
+ * Function that returns a tutorial if it can be found, otherwise returns null.
```

Comment:
Uber nitpicking:
Good practice is to avoid saying "Function that returns [...]" and just say "Returns [...]". This is because you should not need to repeat the nature/type of the commented thing, because it should be obvious.